retrieval/discovery/kubernetes/kubernetes.go

```diff
@@ -35,8 +36,40 @@ const (
 	// in this discovery.
 	metaLabelPrefix = model.MetaLabelPrefix + "kubernetes_"
 	namespaceLabel  = metaLabelPrefix + "namespace"
+
+	// Constants for instrumentation.
+	namespace = "prometheus"
+)
+
+var (
+	eventCount = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "sd_kubernetes_events_total",
+			Help:      "The number of Kubernetes events received.",
+		},
+		[]string{"type", "event"},
```

Comment:
The label name "type" should be avoided, as it's very generic. "role" would be a good choice here.